### ISSUE TITLE
makeup-menu-button: add aria-haspopup if missing

### ISSUE THREAD

**Issue by ianmcburnie** [makeup/makeup-js#91]
I noticed the MIND pattern examples are missing aria-haspopup ,whereas the makeup example isn't. This is because Makeup example is adding the attribute in the base markup:

https://github.com/makeup/makeup-js/blob/master/docs/makeup-menu-button/index.html#L29

The `aria-haspopup` attribute should be added programmatically, if not already included in the base markup.
